superset-frontend/cypress-base/cypress/e2e/dashboard/nativeFilters.test.ts

```diff
@@ -428,14 +428,13 @@ describe('Native filters', () => {
         cy.contains(testItems.filterDefaultValue).should('be.visible');
         cy.contains(testItems.filterOtherCountry).should('not.exist');
       });
+      validateFilterContentOnDashboard(testItems.filterDefaultValue);
 
       // reload dashboard
       cy.reload();
       cy.get(dataTestChartName(testItems.topTenChart.name)).within(() => {
         cy.contains(testItems.filterDefaultValue).should('be.visible');
-        cy.contains(testItems.filterOtherCountry).should('not.exist');
```

Comment:
`cy.reload()` in Cypress 13 seems to completely remove local cache for stored filter inputs. Lmk if my change here is a fix or a band-aid :D